### PR TITLE
adding mount mode for volumes

### PR DESCRIPTION
**Pull request by srgrn**
I had a problem since i mounted some volumes as readonly in my compose files 
so the mount line was 
- src:dst:ro

this commit actually makes all volumes to be mounted as readwrite in a verbose way by adding readonly=0 for all volumes and readonly=1 for the ro volumes

### CODE REVIEW

File: docker_compose_swarm_mode.py [hunk 1]
```diff
@@ -141,15 +141,19 @@ def networks():
 
                 def volumes():
                     for volume in value:
-                        src, dst = volume.split(':')
-
+                        splitted_volume = volume.split(':')
+                        src = splitted_volume.pop(0)
+                        dst = splitted_volume.pop(0)
+                        mode = 0
```
Comment (ddrozdov) [inline docker_compose_swarm_mode.py]: Let's call this variable `readonly`.

File: docker_compose_swarm_mode.py [hunk 2]
```diff
@@ -141,15 +141,19 @@ def networks():
...
+                        if a and a[0] == 'ro':
```
Comment (ddrozdov) [inline docker_compose_swarm_mode.py]: What is `a`? Should it be `splitted_volume` instead?

**Review by ddrozdov — CHANGES_REQUESTED** [review]

**Comment by srgrn** [comment]
Sorry I reverted a change after checking. 
Ill fix it in a few minutes

**Comment by srgrn** [comment]
I have fixed the bugs and also added support for CMD in array format so i want to open a new pull request

**Pull request closed without merging**